spec/support/vcr.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+VCR.configure do |c|
+  c.cassette_library_dir = 'spec/fixtures/vcr_cassettes'
+  c.hook_into :webmock
+  c.configure_rspec_metadata!
+
+  c.filter_sensitive_data('<ACCESS_TOKEN>') { ENV['ACCESS_TOKEN'] }
+  c.filter_sensitive_data('<API_BASE_URI>') { ENV['API_BASE_URI'] }
```

Comment:
don't think base_uri is sensitive data

Reply:
Not sensitive.
My rationale came after asking the question - "Why have a client specific env URL in code?"

Reply:
VCR dependency is removed along with this file in https://github.com/nebulab/solidus_pay_tomorrow/pull/3/commits/59d028bad7d0e18d9404ccfff023ed20cc3a5f80